.github/workflows/pack.yml

```diff
@@ -0,0 +1,99 @@
+name: Pack
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
```

Comment:
```suggestion
```

Otherwise this uploads new binaries (to the latest release I think) on ever PR

Reply:
Sure, will remove once it fully succeeds once here